index.js

```diff
@@ -70,14 +71,20 @@ function invokeCommand(argv) {
         logger.verbose(`Multiple instances found. Skipping passing command to ${pkgName}`);
       } else {
         logger.verbose(`Passing command to ${pkgName}`);
-        module.runCommand(command, argv);
+
         modulesCalled[pkgName] = true;
+        if (module.runCommand(command, argv)) {
+          modulesAnswered.push(pkgName);
```

Comment:
I see you're passing the package name into the `modulesAnswered` array. Would it be good to print those packages on line 87? If not, couldn't this be a simple counter?

Reply:
Good suggestion @Blackbaud-SteveBrush.  Updated to log those out as well.